rbac/management/role/view.py

```diff
@@ -371,7 +394,7 @@ def update(self, request, *args, **kwargs):
         @apiParam (Path) {String} id Role unique identifier
 
         @apiParam (Request Body) {String} name Role name
-        @apiParam (Request Body) {Array} access Access definition
+        @apiParam (Request Body) {ArRray} access Access definition
```

Comment:
Might be a small nitpick not sure if the docstrings affect things here but should `ArRray` just be `Array`?